test/statetest/statetest_loader.cpp

```diff
@@ -69,19 +65,89 @@ state::AccessList from_json<state::AccessList>(const json::json& j)
     return o;
 }
 
+inline uint64_t calculate_current_base_fee_eip1559(
+    uint64_t parent_gas_used, uint64_t parent_gas_limit, uint64_t parent_base_fee)
+{
+    static const auto BASE_FEE_MAX_CHANGE_DENOMINATOR = 8;
+    static const auto ELASTICITY_MULTIPLIER = 2;
+
+    uint64_t base_fee = 0;
```

Comment:
Is 64-bit precision enough for these? Someone should double check EIP-1559.

Reply:
TODO added.